nepal_compliance/api/api.py

```
@@ -16,40 +16,23 @@ def attendance(employee, status, shift, attendance_date, log_type, datetime, lat
+            attendance_doc = frappe.new_doc("Attendance")
+            attendance_doc.employee = employee
+            attendance_doc.status = status
+            attendance_doc.shift = shift
+            attendance_doc.attendance_date = attendance_date
+            attendance_doc.docstatus = 1
+            attendance_doc.insert()
+            attendance_doc_name = attendance_doc.name
+        if attendance_doc_name:
+            employee_check_in.db_set("attendance", attendance_doc_name)
         frappe.db.commit()
     except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)